docs/docsite/rst/reference_appendices/YAMLSyntax.rst

```diff
@@ -107,7 +107,10 @@ While in the above ``>`` example all newlines are folded into spaces, there are
         d
           e
         f
-    same_as: "a b\nc d\n  e\nf\n"
```

Comment:
this is intended as a variable, to show multiple ways to define them, changing it out into a heading changes the significance of the example, if the problem is 'same_as', it can be renamed to  `no_fold_same_newlines:`

Reply:
@bcoca  Thanks for the feedback. I understood the intend here and added a commit to maintain the intend as well to clarify ambiguity